pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -56,6 +61,7 @@ func NewCmdIPFailoverConfig(f *clientcmd.Factory, parentName, name string, out,
 			ErrOut: errout,
 		},
 		ImageTemplate:    variable.NewDefaultImageTemplate(),
+		ServiceAccount:   "ipfailover",
```

Comment:
I also added the creation of the service account object to ipfailover  see [ipfailover.go](https://github.com/JacobTanenbaum/origin/blob/7276b5d79e659d687adc62b5a1b65e2b7dbc8383/pkg/cmd/experimental/ipfailover/ipfailover.go#L190), is that beyond the scope of this fix? Or do you want the doc change ready with this merge?